test-resources/errors/ca-repo-response-invalid-xml.json

```diff
@@ -1 +1 @@
-{"label":"ca-repo-response-invalid-xml","msg":"CA 'ca' got invalid repository response xml: expected some tag","args":{"ca":"ca","cause":"expected some tag"}}
\ No newline at end of file
+{"label":"ca-repo-response-invalid-xml","msg":"CA 'ca' got invalid repository response: expected some tag","args":{"ca":"ca","cause":"expected some tag"}}
```

Comment:
Same comment as above, i.e. you're removing XML from the string content but not from the translation key name.